src/app/src/App.tsx

```diff
@@ -3,33 +3,23 @@ import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
 import './App.css'
 
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet'
+import Map from './Map/Map'
+
+
+
 function App() {
   const [count, setCount] = useState(0)
 
   return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+    <div>
+      <div className="title">THIRD PLACES PROJECT</div>
+      <div className='leaflet-container'>
+        <Map/>
+      </div> 
```

Comment:
I think for organization's sake, this div can go inside the Map component (wrapping the MapContainer component in the return) rather than having it here. Then all the map code will be contained there.